packages/core/src/test/credentials/auth.test.ts

```diff
@@ -256,15 +256,10 @@ describe('Auth', function () {
             assert.strictEqual(auth.getConnectionState(conn), 'valid')
         })
 
-        it('reauthentication is indicated in metric', async function () {
+        it('reauthentication flag is set at start of reauth process', async function () {
             const conn = await auth.createInvalidSsoConnection(ssoProfile)
             await auth.reauthenticate(conn)
-            assertTelemetry('aws_loginWithBrowser', {
-                result: 'Succeeded',
-                isReAuth: true,
-                credentialStartUrl: ssoProfile.startUrl,
-            })
-            assert.strictEqual(getMetrics('aws_loginWithBrowser').length, 1)
+            auth.getTestTokenProvider(conn).createToken.calledWith({ isReAuth: true })
```

Comment:
can't we test the actual metric, as before?

Reply:
This suite already stubs the underlying sso token provider `createToken()` in the test so no telemetry calls are actually made in this case.

This test only existed due to the `aws_loginWithBrowser` hack we previously used in `auth.ts`. Now that all telemetry is isolated to `SsoAccessTokenProvider.ts` telemetry is asserted in the SsoTokenProvider tests.